### DISCUSSION TITLE
Donde subir el repositorio de practicas de semana 1

### DISCUSSION THREAD

**Discussion by vicogarcia16** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#119] [Q&A] [answered]
Buenas noches 
No puede escuchar si hoy comentaron algo referente al formulario, para subir el repositorio con las practicas de la semana 1. Alguien puede comentar?

**Comment by GloriaCoralCerecedo** [comment]
Mañana lo va a subir

**Comment by romarpla** [comment, marked as answer]
[Link de publicación de Teams](https://teams.microsoft.com/l/message/19:8a61b3259c3147259ba0197e9b103432@thread.tacv2/1645814720628?tenantId=4ae54b05-b77e-4224-aef1-8661422e0816&groupId=d38ceaac-3599-4f30-94ba-740ccb530cb0&parentMessageId=1645814720628&teamName=Misi%C3%B3n%20Frontend%20-%20JS&channelName=Semana%201%20-%20Despegue%20%F0%9F%9A%80&createdTime=1645814720628)

Este es el link dentro del equipo de Teams para que puedan ver el link del Formulario
